example/config/.mocharc.yml

```diff
@@ -44,3 +44,8 @@ trace-warnings: true # node flags ok
 ui: bdd
 v8-stack-trace-limit: 100 # V8 flags are prepended with "v8-"
 watch: false
+watchFiles:
```

Comment:
same

Reply:
Fixed in https://github.com/mochajs/mocha/pull/3980/commits/f18a0b66b5df455fc7b1598797e0e3245df52826